castle/context/default.py

```diff
@@ -31,6 +31,14 @@ def defaults_extra(self):
         return context
 
     def call(self):
-        context = dict(self.defaults())
-        context.update(self.defaults_extra())
+        context = dict(self._defaults())
+        context.update(self._defaults_extra())
         return context
+
+    @staticmethod
+    def _fetch_cookies(request, cookies):
+        if cookies:
+            return cookies
```

Comment:
the previous behavior was to use the passed in cookies only if the request cookies didn't exist. this doesn't feel like a bad change, just wanted to call it out.

Reply:
the behavior is opposite. to prevent reading from the request you can pass similar information as an argument (just in case there will be some malformed or different kind of request object there)